infrastructure/zk/src/run/forced-exit.ts

```diff
@@ -0,0 +1,129 @@
+import { Command } from 'commander';
+import fetch from 'node-fetch';
+import * as utils from '../utils';
+import { BigNumber, BigNumberish, ethers } from 'ethers';
+
+type State = {
+    balances: {
+        [token: string]: BigNumberish;
+    };
+    nonce: number;
+    pubKeyHash: string;
+};
+
+type RestAccountState = {
+    finalized: State | undefined;
+};
+
+type RpcAccountState = {
+    verified: State;
+};
+
+async function checkForcedExitSenderAccountBalance(nodeUrl: string, nodeType: 'REST' | 'JSONRPC') {
+    const forcedExitAccount = process.env.FORCED_EXIT_REQUESTS_SENDER_ACCOUNT_ADDRESS as string;
+    let state: State | undefined;
+    if (nodeType === 'REST') {
+        ({ finalized: state } = await processRestRequest(nodeUrl, forcedExitAccount));
+    } else {
+        ({ verified: state } = await processJsonRpcRequest(nodeUrl, forcedExitAccount));
+    }
+
+    if (!state || state.pubKeyHash === 'sync:0000000000000000000000000000000000000000') {
+        if (state?.balances['RBTC']) {
+            const balance = BigNumber.from(state.balances['RBTC']);
+            if (!balance.isZero()) {
+                console.log(`Forced exit sender account balance is ${balance.toString()} RBTC`);
+                console.log('Wait until the preparation of the forced exit sender account is completed');
+            }
+        }
```

Comment:
Sorry for being pedantic, but I find this if statements a bit confusing. What do you think if we change them to be more explicit?
```suggestion
if (!state) {
// cannot retrieve the state
return;
} 
if ( state.pubKeyHash !== 'sync:0000000000000000000000000000000000000000') {
// account is ready
return
}
if ( state?.balances['RBTC']) {
// check the balance
}

```

Reply:
```typescript
if (!state) {
    // cannot retrieve the state
return;
```
The null state does not mean that cannot retrieve the state, it means that the state it is 0, but the REST service, send it as null.

Reply:
But I get what you mean, Ill update it.

Reply:
updated